tasks/views.py

```diff
@@ -16,28 +16,37 @@ def view_tasks(request):
 
 
 def new_task(request):
-    if request.method == "POST":
-        form = TaskForm(request.user.id, request.POST)
-        taskForm = form.save(commit=False)
-        if form.is_valid():
-            try:
-                Task.create_task(taskForm.title, taskForm.created_by, taskForm.assignee, taskForm.priority,
-                                 taskForm.status, taskForm.description)
-
-                messages.success(request, 'Task was added successfully')
+    context = {'manager': False}
+    if not request.user.is_anonymous:
+        logged_user = User.objects.get(user=request.user)
+        if logged_user is not None:
+            context = {'user': logged_user}
+        if logged_user.is_manager():
```

Comment:
In general, you are correct but since I checked a line before if the user is not anonymous this condition is always TRUE.

....I removed the condition